rotkehlchen/tests/unit/test_evm_names.py

```diff
@@ -105,3 +112,41 @@ def test_uses_sources_only_when_needed(evm_address, database: 'DBHandler'):
         )],
     )
     assert names == [], 'No names should have been returned since the blockchain was None'
+
+
+@freeze_time('2023-03-20')  # freesing time just to make sure comparisons of timestamps won't fail
+def test_download_ens_avatar(ethereum_inquirer):
```

Comment:
Can't you also vcr this? I think it should be VCRed as it will be making network calls.

And `random.ens.name.eth` can at some point in the future add an avatar.
And `nebolax.eth` can remove his :wink:

Reply:
Yes, makes sense. I thought that vcr doesn't support binary requests/responses, but it seems like it does.